### ISSUE TITLE
Need to add .rtf conversion support

### ISSUE THREAD

**Issue by kbenoit** [kbenoit/readtext#90]
Write a package to wrap around http://www.gnu.org/software/unrtf/?

**Comment by jeroen** [comment]
Can you try https://github.com/ropensci/unrtf and see if this works?

**Comment by kbenoit** [comment]
Looks great except the encoding issue:
```
Here are some special characters: �t �rv�ztqrQ �tvef�r�g�p, which means five flood resistant hammer drills (J) in Hungarian.
```

**Comment by jeroen** [comment]
Yes I noticed this as well. Not sure if this is a bug on my side or just a deficiency in the `unrtf` utility.

**Comment by jeroen** [comment]
If you specify output format `html` it *does* seem to work, so that makes me think this is a bug in unrtf.

**Comment by jeroen** [comment]
@kbenoit pushed a fix for the encoding thing. Can you check again? If you have additional test cases that would be great too :)

**Comment by kbenoit** [comment]
yes just opened an issue
